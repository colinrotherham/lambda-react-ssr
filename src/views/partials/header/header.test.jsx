import React from 'react'
import renderer from 'react-test-renderer'
import Header from './header'

describe('Partial: Header', () => {
  test('Component is rendered', () => {
    const component = renderer.create(<Header />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
