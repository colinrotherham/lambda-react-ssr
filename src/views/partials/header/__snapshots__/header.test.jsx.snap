// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Partial: Header Component is rendered 1`] = `
<header
  className="header"
>
  <h1>
    Hello header
  </h1>
  <p>
    Yes this is a header
  </p>
</header>
`;
